src/main.ts

```diff
@@ -15,24 +14,17 @@ import store from './store';
 
 import * as fb from './firebaseConfig';
 
-Vue.config.productionTip = false;
-
-Vue.use(BootstrapVue);
```

Comment:
Do we not use bootstrap anywhere on CoursePlan? I had thought the landing page used some bootstrap classes, but could be wrong

Reply:
We only use bootstrap css, but not vue components